.gitignore

```diff
@@ -1,3 +1,4 @@
+launch.json
```

Comment:
@Envl You might know this already, but please be aware of the fact that once a file/folder has been added to the Git project it has to be removed manually before .gitignore will take effect. This can be done by $ git -rm file/folder.